clients/src/main/java/org/apache/kafka/clients/consumer/internals/CommitRequestManager.java

```diff
@@ -93,9 +94,11 @@ public CommitRequestManager(
             final SubscriptionState subscriptions,
             final ConsumerConfig config,
             final CoordinatorRequestManager coordinatorRequestManager,
+            final OffsetCommitCallbackInvoker offsetCommitCallbackInvoker,
             final String groupId,
             final Optional<String> groupInstanceId) {
-        this(time, logContext, subscriptions, config, coordinatorRequestManager, groupId,
+        this(time, logContext, subscriptions, config, coordinatorRequestManager,
```

Comment:
i wonder if it would be more aesthetic to split each of them into its own line.... now it is spanning 3 lines with different widths.

Reply:
Done